woocommerce/woocommerce-android
# [Manual Taxes M#2] Tax rate selector: UI part 3

### [Manual Taxes M#2] Tax rate selector: UI part 3
This PR brings:
* Feature flag
* "Set New Tax Rate" button in the order creation screen (hidden under the feature flag)
* Toolbar in the tax rate selector screen
* Removal of ">" icon in tax rate rows in tax rate selector screen (as per discussion: p1693793634295599/1693400109.681069-slack-C0354HSNUJH)

### Testing instructions

1. In the order creation screen verify that the "Set New Tax Rate" button is visible only in debug
2. Click on the "Set New Tax Rate" button
3. Verify the tax rate selector opens and shows three mock items
4. Verify tax rate selector has a toolbar

### Images/gif

| Dark | Light |
|---|---|
| <img width=300 src=https://github.com/woocommerce/woocommerce-android/assets/4527432/e8b6585f-3491-42bb-9a1c-898123acdb51/> | <img width=300 src=https://github.com/woocommerce/woocommerce-android/assets/4527432/5192a845-b4d5-4cef-b361-fd27955d83ba/> |
| <img width=300 src=https://github.com/woocommerce/woocommerce-android/assets/4527432/b451a0be-fe26-44f2-a363-3d9f7746cbe6/> | <img width=300 src=https://github.com/woocommerce/woocommerce-android/assets/4527432/c192c83e-b64a-4c39-8ec0-e7a36fa62468/> |
